src/Altinn.App.PlatformServices/Altinn.App.PlatformServices.csproj

```diff
@@ -1,21 +1,17 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net5.0</TargetFramework>
+    <TargetFrameworks>netcoreapp3.1;net5.0;net6.0</TargetFrameworks>
```

Comment:
Do we need netcoreapp3.1?

Reply:
Did not support netcoreapp3.1 so it's removed